docs/archive/project-json.md

```diff
@@ -42,15 +42,32 @@ The [`project.lock.json`](#projectlockjson) file (described below) is also used
 
 ## Migrate project.json to PackageReference
 
-The migration between project.json and PackageReference is straightforward. The easiest way to do it to use the built-in migrator in the latest Visual Studio 2022, Update 14.
+The migration between project.json and PackageReference is straightforward.
+
+### Automatic migration in Visual Studio 2026
+
+Visual Studio 2026 and later automatically migrates project.json projects to PackageReference when you open a solution containing project.json projects. The migration happens at solution load time:
+
+1. Open a solution containing project.json projects in Visual Studio 2026 or later.
+1. Visual Studio automatically detects project.json files and migrates them to PackageReference format.
+1. A backup of the original project file and project.json file is created in a `Backup` folder in the root of the project directory.
+1. The migration converts all package dependencies to PackageReference format in the project file.
+
+If you need to roll back the automatic migration, you can restore the original files from the backup folder.
```

Comment:
@nkolev92 How should we explain the Backup folder? It will not be usable in the new tooling.

Reply:
I forget, did we remove the folder or what?
honestly, I'd probably just recommend that people revert the files changes. 
Everyone uses source control nowadays, so not really a concern imo.

Reply:
I see. 
I don't remember the gotcha then, what's your concern with this line? 
Folder aside, I think we can just say revert your project file changes instead. 

Feel free to ping me offline to discuss further if you want.